protocol/0056-REWA-rewards_overview.md

```diff
@@ -51,7 +51,87 @@ These metrics apply only to the sum of fees for the epoch in question.
+- $OI_{n}$ be the parties open interest before their open interest changed
+- $t_{n}$ be the time the party held the previous position in seconds
+- $t$ be the amount of time elapsed in the current epoch so far
+
+
+$$\bar{OI} = \bar{OI} \cdot \left(1 - \frac{t_{n}}{t}\right) + \frac{OI_{n} \cdot t_{n}}{t}$$
+
+At the end of the epoch, the network must store the parties time weighted average open interest and then calculate their open interest reward metric as follows.
+
+Let:
+
+- $m_{oi}$ be the parties open interest reward metric
+- $\bar{OI_{i}}$ be the parties time weighted average open interest in the ith epoch
+- $N$ be the network parameter `rewards.metrics.openInterestWindow`
+
+$$m_{oi} = \frac{\sum_{i}^{n}\bar{OI_{i}}}{N}$$
+
+### Relative return metric
+
+The relative return metric, $m_{rr}$, measures each parties average relative return, weighted by their [time-weighted average open-interest](#open-interest-metric), over a number of epochs.
+
+At the start of the epoch, the network must store the parties current pnl.
+
+$$p_{0} = p_{realised} + p_{unrealised}$$
+
+At the end of each epoch, the network must calculate and store the parties change in pnl:
+
+Let:
+
+- $\Delta{p}$ be the parties change in pnl
+- $p_{realised}$ be the parties realised pnl at the end of the epoch
+- $p_{unrealised}$ be the parties unrealised pnl at the end of the epoch
+
+$$\Delta{p} = p_{realised} + p_{unrealised} - p_{0}$$
+
+And calculate their average relative return, weighted by the log of their [time weighted average open interest](#open-interest-metric), over the last $N$ epochs as follows.
+
+Let:
+
+- $m_{rr}$ be the parties relative return reward metric
+- $\bar{OI_{i}}$ be the parties time weighted average open interest in the ith epoch
+- $\Delta{p}$ be the parties change in pnl in the ith epoch
+- $N$ be the network parameter `rewards.metrics.relativeReturnsWindow`
+
+$$m_{rr} = \frac{\sum_{i}^{n}{$\Delta{p}$\cdot\log(1 + \bar{OI_{i}})}}{N}$$
+
+### Returns volatility metric
+
+The return volatility metric, $m_{rv}$, measures the volatility of a parties returns across a number of epochs.
+
+At the end of an epoch, if a party has had net returns less than or equal to `0` over the last $N$ epochs (where $N$ is the network parameter `rewards.metrics.returnsVolatilityWindow`), their reward metric $m_{rv}$ is set to `0`. Otherwise, the network should calculate the standard deviation of the set of each parties returns weighted by the log of their [time weighted average open interest](#open-interest-metric) over the last $N$ epochs.
+
+Given the set:
+
+$$R = \{r_i \cdot \log(1 +\bar{OI_{i}}) \mid i = 1, 2, \ldots, N\}$$
+
+The reward metric $m_{rv}$ is the standard deviation of the set $R$.
+
+### Validator power metric
+
+The validator power metric, $m_{vp}$, measures the voting power of validators.
+
+At the end of each epoch, for each party who is a validator set their reward metric as follows.
+
+$$m_{vp} = validatorScore$$
```

Comment:
what validator score? why not just the voting power directly?

Reply:
I generally don't understand how this is supposed to work - if I specify this metrics with no parties in scope, does anyone get it? the way it's written above, does it go only to consensus validators and not ersatz validators? pending validators? those would have non negative score (by some definition of score) but would have no voting power so not sure what the intention is. If I don't qualify parties, who gets the reward? no one? all validators ranked by their score?